test/SlowTests/Server/Documents/PeriodicBackup/Ftp.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentFTP;
+using FubarDev.FtpServer;
+using FubarDev.FtpServer.FileSystem.InMemory;
+using Microsoft.Extensions.DependencyInjection;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Documents.Operations.Backups;
+using Raven.Server.Documents.PeriodicBackup;
+using Raven.Tests.Core.Utils.Entities;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Server.Documents.PeriodicBackup
+{
+    public class Ftp : CloudBackupTestBase
```

Comment:
assuming that those tests with the old client were passing as well?

Reply:
Correct, with a bit of change because of the difference between the 2 libraries (FluentFTP and FTPWebRequest)